community/entr/APKBUILD

```diff
@@ -1,15 +1,18 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=entr
-pkgver=3.6
-_hgrev=c15b0be493fc
+pkgver=3.7
+_hgrev=c5b62bde107d
```

Comment:
Please avoid this, it complicates bumping of the pkg and it’s actually unnecessary. You can workaround this problem with hash in the directory with something like:

```sh
builddir="$srcdir/eradman-$pkgname"

prepare() {
	# Top-level directory inside the tar archive contains SHA of a commit...
	cd "$srcdir"
	mv $pkgname-v$pkgver-* "$builddir"

	default_prepare
}
```